### ISSUE TITLE
Add a simple example to the root module

### ISSUE THREAD

**Issue by DevQps** [NLnetLabs/rpki-rs#33]
## Description
Currently, there is no example given in the root module which makes it very hard to figure out how to actually use this library. It seems very feature-rich so I am sure many people would be glad to use it if they get some directions on how they should approach it!

If I can be of any help during this process please let me know! I am not very sure on how to use this crate yet but maybe I am able to figure it out with a few hints :)

Some examples that come to my mind:
- Downloading / syncing ROA's with TALs (not sure if this is outside the scope of this crate?)
- Matching against all ROA's.

**Comment by partim** [comment]
You are correct that documentation is rather lacking. I am improving it here and there, but indeed there must be better top-level documentation.

That said, the crate is intended to be rather low-level. It contains the functionality that is shared between [Routinator](https://github.com/NLnetLabs/routinator) and [Krill](https://github.com/NLnetLabs/krill) (i.e., the relying party and CA software, respectively). It really only provides the building blocks to start something useful from. Your examples, which rely on having a local copy of the RPKI repository available, are probably better achieved by using Routinator as a library crate – with the caveat that its documentation is in an even worse state.

**Comment by DevQps** [comment]
@partim Thanks for your response! I will go check that out then :) If there's anything I can do to help, please let me know.
